server/src/rooms/data/roomData.json

```diff
@@ -185,10 +185,10 @@
     "roomId": "LoversLake"
   },
   "screen3": {
-    "displayName": "Screen 3",
-    "shortName": "Screen 3",
+    "displayName": "Screen 3: Jabberwock Park",
+    "shortName": "Jabberwock Park",
     "id": "screen3",
-    "description": "You enter a theater projecting an action movie of some kind. Explosions fill the room with light and the sound rocks the flat surfaces so intensely that you are nearly knocked off your feet. An errant popcorn kernel hits the side of your head in time with sharp bark of a scream, causing you to involuntarily flinch.<br/><br/>Or you can go back to the [[main theater->theater]].",
+    "description": "You enter a pitch black space and immediately a roaring mouth fills the screen. A vorpal sword shifts into and then out of focus as the camera focuses on a mirror which reads \"Objects in mirror are farther away than they appear\".<br/><br/>Or you can go back to the [[main theater->theater]].",
```

Comment:
A vorpal sword shifts into and then out of focus as the camera zooms in on a mirror which reads \"Objects in mirror are farther away than they appear\".<br/><br/>You can take a seat here, or you can go back to the [[main theater->theater]].